### PR TITLE
Add change over endpoint

### PR DESCRIPTION
**Pull request by JulienItard**

### CODE REVIEW

File: api_calls/reference/change_overs/public-scope-response.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    {
+      "links": {
+        "rental": 2
+      },
+      "id": 2,
+      "map": "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
```
Comment (Azdaroth) [inline api_calls/reference/change_overs/public-scope-response.json]: A better example would be useful, with all possible values.

File: api_calls/reference/change_overs/public-scope-response.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      "start_date": "START_DATE",
```
Comment (Azdaroth) [inline api_calls/reference/change_overs/public-scope-response.json]: Too much copy & paste :doge:

File: content/reference/endpoints/change_overs.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | ChangeOver's id.
+-----------------|---------|------------|------------
+map              | String  | Read       | ChangeOver's map of booking status. There is a total of 1096 characters, each representing one day. List of statuses described in [enums section](/reference/enums#change_over-map-statuses).
```
Comment (Azdaroth) [inline content/reference/endpoints/change_overs.md]: It's not really a booking status, more like, hmm, a map of possible actions for a given day?

File: content/reference/endpoints/change_overs.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+-----------------|---------|------------|------------
+start_date       | [Date](/reference/enums#formats) | Read       | ChangeOver's start date.
+updated_at       | [Time](/reference/enums#formats) | Read       | ChangeOver's update time.
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List change_overs
+
+List all change_overs for given account.
```
Comment (Azdaroth) [inline content/reference/endpoints/change_overs.md]: `for a given account` #grammarNazi

Comment (JulienItard) [reply content/reference/endpoints/change_overs.md]: There is no `a` in all endpoint 🐶 Are you sure ?

Comment (Azdaroth) [reply content/reference/endpoints/change_overs.md]: Well, it means it's wrong everywhere :D

File: api_calls/reference/change_overs/public-scope-response.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      "links": {
+        "rental": 2
+      },
+      "id": 2,
+      "map": "CCOOOOCCCIIIIIIIIICCCCCCCCCCCXXXXCCCCCCCCCCCCCCCCCCCCCCCCCCCXXXXXXXXCCCCCCCCCCCCCCCCCCCCOOOOOOIIIIXXXCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+      "start_date": "START_DATE",
```
Comment (Azdaroth) [inline api_calls/reference/change_overs/public-scope-response.json]: How about providing some real values?

**Pull request merged**